### DISCUSSION TITLE
Management PipeCD v0.44.2 features

### DISCUSSION THREAD

**Discussion by khanhtc1202** [pipe-cd/pipecd#4443] [General] [answered]
About PipeCD release cycle: [Release docs](https://github.com/pipe-cd/pipecd/blob/master/RELEASES.md#release-cycle)
v0.44.2 project board: [v0.44.2](https://github.com/orgs/pipe-cd/projects/3/views/1)

Any other ideas/requests are more than welcome. Feel free to add as comments under this discussion 🙏

**Comment by kentakozuka** [comment, marked as answer]
https://github.com/pipe-cd/pipecd/pull/4469
https://github.com/pipe-cd/pipecd/pull/4465
https://github.com/pipe-cd/pipecd/pull/4466
https://github.com/pipe-cd/pipecd/pull/4463
https://github.com/pipe-cd/pipecd/pull/4464
https://github.com/pipe-cd/pipecd/pull/4462
https://github.com/pipe-cd/pipecd/pull/4457
https://github.com/pipe-cd/pipecd/pull/4451
